packages/core/src/bus/Command.ts

```diff
@@ -0,0 +1,39 @@
+import { CommandDispatcher } from './CommandDispatcher';
+import { getTypeName } from '../utils';
+import { v4 as uuidv4 } from 'uuid';
+
+export abstract class Command<T, R> {
+  public readonly expectReply = true;
+  public readonly ttl = 10000;
+  public readonly type: string;
+  public readonly payload!: T;
+  public readonly correlationId!: string;
+  public readonly createdAt!: Date;
+
+  protected constructor(
+    payload: T,
+    expectReply?: boolean,
+    ttl?: number,
+    type?: string,
+    correlationId?: string,
+    createdAt?: Date
+  ) {
+    this.payload = payload;
+
+    if (expectReply) {
+      this.expectReply = expectReply;
+    }
```

Comment:
It makes it impossible to set `false`

```suggestion
    if (typeof expectReply === 'boolean') {
      this.expectReply = expectReply;
    }
```